client/src/listings/Listings.js

```diff
@@ -196,7 +197,7 @@ function Listings(props) {
               autoFocus
             />
           </Grid>
-          <Grid item xs={3}>
+ {/*          <Grid item xs={3}>
```

Comment:
Generally, it's better to outright delete things we aren't using rather than commenting them. They are saving forever in our Git version history if we ever need to retrieve them. :)